GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.cs

```diff
@@ -50,6 +53,7 @@ protected override void SettingsToPage()
             chkShowCurrentBranchInVisualStudio.Checked = AppSettings.ShowCurrentBranchInVisualStudio;
             _NO_TRANSLATE_DaysToCacheImages.Value = AppSettings.AvatarImageCacheDays;
             ShowAuthorAvatar.Checked = AppSettings.ShowAuthorAvatarInCommitInfo;
+            ShowAuthorAvatarInCommitGraph.Checked = AppSettings.ShowAuthorAvatarColumn;
```

Comment:
I would have preferred that this setting was needed here, but this PR is an improvement. 
A tooltip for the CommitInfo setting is an alternative

Reply:
Apologies, I don't understand what you're saying

Reply:
Related: This setting should have been called ShowAuthorAvatarInRevisionGraph

The columns in the revision graph is normally not set from settings. The existing (now renamed) ShowAuthorAvatarInCommitInfo should have been moved out of here. A "block all avatars" would have been cleaner.

I do not request changes though